packages/react/src/lib/felaInvokeKeyframesPlugin.ts

```diff
@@ -25,6 +29,13 @@ export default () => {
         }
       }
 
+      if (Array.isArray(cssPropertyValue)) {
+        return {
+          ...acc,
+          [cssPropertyName]: cssPropertyValue.map(arrayElement => invokeKeyframes(arrayElement)),
+        }
+      }
```

Comment:
Why it fails and still is not safe?

```tsx
typeof [] // 'object'
typeof null // 'object'
```

I suggest to use `_.isPlainObject` instead:
```tsx
      if (_.isPlainObject(cssPropertyValue)) {
        if (cssPropertyName === 'animationName') {
          if (cssPropertyValue.keyframe) {
            styles[cssPropertyName] = callable(cssPropertyValue.keyframe)(
              cssPropertyValue.params || {},
            )
          }

          return {
            ...acc,
            [cssPropertyName]: styles[cssPropertyName],
          }
        }

        return {
          ...acc,
          [cssPropertyName]: invokeKeyframes(cssPropertyValue),
        }
      }
```